_posts/2022-07-28-where-are-they-now-matt-arnold.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Where are they now? Matt Arnold
+description: We catch up with ex-competitor Matt Arnold.
+---
+
+This week we’ll be catching up with Matt, a Computer Science teacher at Barton Peveril College. Matt competed in SR2014 as a part of team BPV from Barton Peveril College. He went on to study Computer Science at Cambridge University before returning to Barton Peveril as a teacher.
+
+#### How did you first get involved in (competing with) Student Robotics?
```

Comment:
Do we need "competing with" for a non volunteer?

Reply:
```suggestion
#### How did you first get involved with Student Robotics?
```